### ISSUE TITLE
Package Warning

### ISSUE THREAD

**Issue by rbarnett73** [mclemente/fvtt-advanced-macros#75]
module "advanced-macros" uses pack.private which has been replaced with pack.ownership

**Comment by drakonik2782** [comment]
For anyone wanting to remove this error manually, update the manifest like so.
Remove the private declaration:
```
    "packs": [
        {
            "name": "macros",
            "label": "Advanced Macros",
            "path": "packs/macros.db",
            "private": true, // DELETE THIS LINE
            "type": "Macro"
        }
    ],
```
Add this somewhere in the document, perhaps just after `"packs": [ ... ],"`:
```
    "ownership": {
        "PLAYER": "LIMITED",
        "ASSISTANT": "OWNER"
    },
```
Alternatively, if you don't want standard players seeing these macros, you could instead do:
```
    "ownership": {
        "PLAYER": "NONE",
        "TRUSTED": "LIMITED",
        "ASSISTANT": "OWNER"
    },
```

**Comment by mclemente** [comment]
This was fixed v2.0.3 https://github.com/mclemente/fvtt-advanced-macros/commit/ff008d6155c4ed4478303496f2e098ab7ff0c29b
